front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -279,15 +279,17 @@ const TimetableTrainCard = ({
         </div>
 
         <div className="scenario-timetable-train-buttons">
-          <button
-            className="scenario-timetable-train-buttons-selectprojection"
-            type="button"
-            aria-label={t('timetable.choosePath')}
-            title={t('timetable.choosePath')}
-            onClick={selectPathProjection}
-          >
-            <GiPathDistance />
-          </button>
+          {train.isValid && (
```

Comment:
Maybe you can just use disable property on the button ?
It will keep the layout consistent

Reply:
I tried both, but I prefered to hide the button completely. Maybe we can ask Thibaut its thoughts about it ?

Reply:
I created this issue to handle this enhancement, I remove my second commit https://github.com/OpenRailAssociation/osrd/issues/8922